### PR TITLE
feat: #1268 aries-js-worker - parametrize assets path

### PR DESCRIPTION
**Pull request by llorllale**
Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

File: cmd/aries-js-worker/src/aries.js [hunk 1]
```diff
@@ -99,6 +99,24 @@ export const Aries = function(opts) {
...
+    if (!opts.assetsPath) {
+        throw new Error("aries: missing assets path")
+    }
+
+    WORKER = loadWorker(
+        PENDING,
+        NOTIFICATIONS,
+        {
+            dir: opts.assetsPath,
+            wasm: opts.assetsPath + "/aries-js-worker.wasm.gz",
```
Comment (troyronda) [inline cmd/aries-js-worker/src/aries.js]: we should also allow configuration of whether or not the wasm was pre-compressed.
some servers prefer uncompressed and then they returned the compressed encoding as needed.

Comment (llorllale) [reply cmd/aries-js-worker/src/aries.js]: new issue?

Comment (troyronda) [reply cmd/aries-js-worker/src/aries.js]: sure.

(the other idea would be to allow customization of the filename including extension).

Comment (troyronda) [reply cmd/aries-js-worker/src/aries.js]: #1277

**Pull request merged**